cmd/main.go

```diff
@@ -242,8 +244,73 @@ func main() {
 		}
 	}
 
-	nimState := os.Getenv("NIM_STATE")
 	signalHandlerCtx := ctrl.SetupSignalHandler()
+	setupNim(mgr, signalHandlerCtx, kubeClient)
+
+	// nolint:goconst
+	if os.Getenv(enableWebhooksEnv) != "false" {
```

Comment:
we have 3 `if` doing the same thing, maybe it would be better to move all the webhooks under only one `if` statement, wdyt?

Reply:
yea I agree

Reply:
`// nolint:goconst `
I think this should be removed if we gather this to 1 if

Reply:
We have only one `if` :-)